api/v1alpha1/platformoperator_types.go

```diff
@@ -22,9 +22,11 @@ import (
 
 // PlatformOperatorSpec defines the desired state of PlatformOperator
 type PlatformOperatorSpec struct {
-	// PackageName specifies the name of the package to be installed from the provided CatalogSource.
-	// PackageName is required and must equal the exact name of the package in the catalog.
-	PackageName string `json:"packageName"`
+	// Package specifies the name of the package to be installed from the provided CatalogSource.
+	// Package is required and must equal the exact name of the package in the catalog.
+	Package string `json:"package"`
```

Comment:
This can be in a separate PR, but I was thinking maybe we want to add additional kubebuilder markers for these fields. For example, even though they are required, `""` is still technically a valid value for these fields and would pass validation. Maybe we could add something like 

```
+kubebuilder:validation:MinLength=1
```

to ensure that some kind of value must be provided? (Go treats the length of `""` as zero per https://go.dev/play/p/63fGgUxvFOm)